environments/daml-sdk-env/build/3_configsdk.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+echo "PATH=\$PATH:\$HOME/.daml/bin" >> ~/.bashrc
+source ~/.bashrc
+
+# uninstall unnecessary VSCode extensions
+rm -rf /opt/.katacodacode/extensions/redhat.vscode-openshift-connector-0.0.19
+
+# install the daml extension (the hard way)
+cd /tmp & mkdir daml-bundled
+unzip $(find ~/.daml/sdk -d 1)/studio/daml-bundled.vsix -d daml-bundled
```

Comment:
I guess catacoda’s vscode server does not have something like `--install-extension`?

Reply:
It does, but it's somewhat custom so I'll start like this and improve once a first version is running.